package.json

```diff
@@ -9,7 +9,7 @@
     "npm": ">=8.0.0 <9.0.0"
   },
   "dependencies": {
-    "@18f/us-federal-holidays": "^3.0.1",
```

Comment:
did you happen to check the use of this package since this is a major upgrade

Reply:
Good point, I should have mentioned that, thanks for asking! The major version upgrade was due to them [dropping support for Node 12 and 14](https://github.com/18F/us-federal-holidays/releases/tag/v4.0.0). It does not look like any other changes were made with that release!